routes/teams.js

```diff
@@ -525,8 +526,16 @@ router.put("/", Utils.ensureAuthenticated, async (req, res) => {
     return;
   }
   let sql = "UPDATE team SET ? WHERE id=?";
+  
+  //UpdateMatchTeamName
+  let updateMatchTeam1Name = "UPDATE `match` mtch JOIN team t ON mtch.team1_id = t.id SET mtch.team1_string = t.name";
```

Comment:
I'm still against updating this data, as the match table shows a historical view of what a team was called. I would much prefer to check and see if the team still exists in the database, and output that name, or use the match table if the team does not exist.